display-brightness-ddcutil@themightydeity.github.com/prefs.js

```diff
@@ -133,15 +129,15 @@ const PrefsWidget = GObject.registerClass({
 );
 
 function init() {
-    ExtensionUtils.initTranslations();
-}
-
-function buildPrefsWidget() {
     const preferences = new PrefsWidget();
     preferences.connect('notify::root', () => {
         const window = preferences.get_root();
-        const headerbar = new Headerbar();
-        window.set_titlebar(headerbar);
+        window.set_default_size(500, 700);
+        window.search_enabled = true;
```

Comment:
After rebasing this PR with latest changes, these properties are no longer respected and translations are not applying.

Reply:
Fixed using the new `fillPreferencesWindow` function.